cms/standard_pages/models.py

```diff
@@ -38,3 +47,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class MethodologyPage(BasePage):  # type: ignore[django-manager-missing]
+    parent_page_types: ClassVar[list[str]] = ["topics.TopicPage"]
+
+    template = "templates/pages/methodology_page.html"
+
+    summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    published_date = models.DateField()
+    last_revised_date = models.DateField(blank=True, null=True)
+
+    contact_details = models.ForeignKey(
+        "core.ContactDetails",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content = StreamField(SectionStoryBlock())
+
+    show_cite_this_page = models.BooleanField(default=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        MultiFieldPanel(
+            [
+                FieldPanel("published_date", icon="calendar-date"),
+                FieldPanel(
+                    "last_revised_date",
+                ),
+                FieldPanel("contact_details"),
+                FieldPanel("show_cite_this_page"),
+            ],
+            heading=_("Metadata"),
+            icon="cog",
+        ),
+        FieldPanel("content", icon="list-ul"),
+        InlinePanel("page_related_pages", label="Related publications"),
```

Comment:
```suggestion
        InlinePanel("page_related_pages", label=_("Related publications")),
```

Reply:
also note that the AC specifically calls out only Analysis pages.
So you will want a different inline class

```
Related Publications

Given I have created an analysis page
When I insert an internal link to Analysis pages within CMS that relate to my methodology
Then I am able to use the hyperlinks to access and view those related publications
```